### ISSUE TITLE
[feature] Infinite session duration

### ISSUE THREAD

**Issue by viincenb** [heavyai/heavydb#343]
Hi, 

We are using Omnisci for BI things and developed our own Nodejs connector using omnisql. We'd like to avoid the session idle error without writing a huge number in the configuration file (`idle-session-duration=9999999`).  
I tried to set `idle-session-duration` to `0` or `-1` but this does not do the trick.

Would you mind adding this feature for the configuration ?

**Comment by alexbaden** [comment]
Hi @devSwoop, 

Let me pass this along to our product management team and get back to you.

**Comment by randyzwitch** [comment]
Agreed, needed to do something like this when using StreamSets and the JDBC driver. Ended up setting to one year's worth of minutes or something similar for the session length.

**Comment by psilvest** [comment]
This issue has been inactive for a while, and the project has changed substantially. If this still occurs on the latest release, please add reproduction steps and environment details.
